### PR TITLE
Simplify describe and describe_open

### PR DESCRIPTION
**Pull request by martindurant**
Following #329 , this would be something like "option 3"; this fails on a couple of tests to do with evaluating argument templates on the server when opening directly. I don't have a good answer for what is the right thing to do yet, so posting this for conversation.

### CODE REVIEW

**Comment by martindurant** [comment]
@danielballan , thought you might have some comments on this, and it ought to be tried with Bluesky, which is so far the only cat I know of so far that is designed for remote.

File: intake/gui/source/description.py [hunk 1]
```diff
@@ -61,12 +61,12 @@ def contents(self):
         """String representation of the source's description"""
         if not self._source:
             return ' ' * 100  # HACK - make sure that area is big
-        contents, warning = self.source._display_content()
+        contents = self.source.describe()
         if 'metadata' in contents and 'plots' in contents['metadata']:
             contents['metadata'].pop('plots')
         if 'args' in contents and 'plots' in contents['args'].get('metadata', {}):
```
Comment (jsignell) [inline intake/gui/source/description.py]: Does this  duplication still happen? Might want to remove otherwise.

**Comment by jsignell** [comment]
Sorry that was a false alarm. This PR seems good.

**Comment by martindurant** [comment]
Will wait until after #331

**Comment by ian-r-rose** [comment]
:+1:

**Pull request merged**